src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/AssessmentOrgsRepository.cs

```diff
@@ -90,36 +93,32 @@ public IEnumerable<Organisation> GetOrganisationsByStandardId(string standardId)
                 throw new ApplicationException($"Failed query organisations by standard id");
             }
 
-            var organisations = results.Documents.Where(x => x.EffectiveFrom.Date <= DateTime.UtcNow.Date);
+            var organisations = results.Documents.Where(x => x.EffectiveFrom.Date <= DateTime.UtcNow.Date && (x.EffectiveTo >= DateTime.UtcNow.Date || x.EffectiveTo == null)).ToList();
```

Comment:
no need to check that, if EffectiveFrom date is null, it will not be indexed, because as you said we need a valid EffectiveFrom date